src/elements/LinkButton/LinkButton.styles.ts

```diff
@@ -10,14 +10,12 @@ export default StyleSheet.create({
 	textContainer: {
 		height: 30,
 		width: 'auto',
-		borderBottomColor: 'white',
 		borderBottomWidth: 3.5,
 	},
 	text: {
 		textAlign: 'center',
 		fontFamily: 'open-sans-bold',
 		fontSize: 20,
-		color: colors.NAVY_BLUE,
 	},
 	pressed: {
```

Comment:
Unused style classes can be removed too :)

Reply:
removed :)